tests/tools/onert_train/src/args.cc

```diff
@@ -267,8 +267,7 @@ void Args::Parse(const int argc, char **argv)
 
   if (vm.count("help"))
   {
-    std::cout << "onert_train\n\n";
-    std::cout << "Usage: " << argv[0] << "[model path] [<options>]\n\n";
+    std::cout << "Usage: " << argv[0] << " [model path] [<options>]\n";
```

Comment:
I would like to maintain the same usage format as `onert_run` to have unity for ONE users. Could you keep this code as it is or modify `onert_run` like this? :)

Reply:
Which format you want to keep?
0. `std::cout << "Usage: " << argv[0] << "[model path] [<options>]\n\n";` (original)
1. `std::cout << "Usage: " << argv[0] << " [model path] [<options>]\n\n";` (add space)
2. `std::cout << "Usage: " << argv[0] << "[model path] [<options>]\n";` (remove one new line)

Reply:
FYI,

from now on wo this PR,
```
$ ./Product/out/bin/onert_train --help
onert_train

Usage: ./Product/out/bin/onert_train[model path] [<options>]
```

Reply:
```
$ ./Product/out/bin/onert_run --help
onert_run

Usage: ./Product/out/bin/onert_run path to nnpackage root directory [<options>]


General options:
```

The current code is the same with `onert_run`.

Reply:
I got it!